examples/cluster/floating_ip.sh

```diff
@@ -0,0 +1,6 @@
+# add ip alias
+/sbin/ip address add $EXTERNAL_IP/24 dev $EXTERNAL_INTERFACE
```

Comment:
perhaps a list of commands to execute as part of this phase, but also this is fine.